angr/engines/unicorn.py

```diff
@@ -155,6 +238,10 @@ def process_successors(self, successors, **kwargs):
                         state._inspect('irsb', BP_AFTER, address=bbl_addr)
                     break
 
+        if state.unicorn.stop_reason in (STOP.symbolic_stop_reasons + STOP.unsupported_reasons) or \
+          state.unicorn.stop_reason in (STOP.STOP_UNKNOWN_MEMORY_WRITE, STOP.STOP_VEX_LIFT_FAILED):
```

Comment:
The second line looks ugly right now. I would suggest following one of the styles in [PEP8](https://www.python.org/dev/peps/pep-0008/#multiline-if-statements).

Reply:
Fixed.